requirements/app/examples.txt

```diff
@@ -0,0 +1,2 @@
+# NOTE: the upper bound for the package version is only set for CI stability, and it is dropped while installing this package
```

Comment:
Remove the file entirely? This is not used in CI

Reply:
dropped